src/models/models/assess.py

```
@@ -271,7 +274,8 @@ class FilterLists(TaranisBaseModel):
 
     tags: list[str] = []
     sources: list[AssessSource] = []
-    groups: list[dict[str, Any]] = []
+    groups: list[dict[str, str]] = []
```

Ruff (0.15.13)

[warning] Mutable default value for class attribute

(RUF012)